app/pulp/app/settings.py

```diff
@@ -165,18 +165,22 @@
             'simple': {'format': 'pulp: %(name)s:%(levelname)s: %(message)s'},
         },
         'handlers': {
-            'stream': {
+            'pulp': {
                 'class': 'logging.StreamHandler',
                 'formatter': 'simple',
             },
         },
         'loggers': {
-            'pulp.app': {
-                'handlers': ['stream'],
+            '': {
```

Comment:
What is the effect of having this be the root logger. Is that the default logger used anytime someone calls: `_logger = logging.getLogger(__name__)`?